cpanfile

```diff
@@ -8,6 +8,7 @@ requires 'Catalyst::Action::RenderView'               => '0.16';
 requires 'Catalyst::Plugin::Authentication'           => '0.10023';
 requires 'Catalyst::Authentication::Credential::HTTP' => '1.016';
 requires 'Catalyst::Plugin::Cache'                    => '0.12';
+requires 'Catalyst::Plugin::Cache::HTTP'              => '0.001000';
```

Comment:
`USE_ETAGS` is enabled by default, so the server would break by default without this.